org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
```

Comment:
As this is a sole contribution by you this line should be:
    2023 DBeaver Corporation